src/QirRuntime/lib/QIR/strings.cpp

```diff
@@ -135,7 +135,10 @@ extern "C"
             return quantum__rt__string_create("PauliY");
         case PauliId_Z:
             return quantum__rt__string_create("PauliZ");
+        default:
+            break;
         }
+        return quantum__rt__string_create("<Unexpected Pauli Value>");

```

Comment:
What does the QIR spec says for this? Shouldn't the call fail rather than return a random string?

Reply:
My impression is that the [QIR Spec](https://github.com/microsoft/qsharp-language/blob/main/Specifications/QIR/Data-Types.md) does not cover that case (I searched through all the occurrences of "pauli").
Thus I needed to invent something.

Reply:
Log a bug against the spec? Imho, it should cover error cases like this.